website/pages/docs/connect/terminating_gateway.mdx

```diff
@@ -0,0 +1,99 @@
+---
+layout: docs
+page_title: Connect - Terminating Gateways
+sidebar_title: Terminating Gateways
+description: >-
+  A Terminating Gateway enables traffic from services in the Consul
+  service mesh to services outside the mesh. This section details
+  how to configure and run a terminating gateway.
+---
+
+# Terminating Gateways
+
+-> **1.8.0+:**  This feature is available in Consul versions 1.8.0 and newer.
+
+Terminating gateways enable connections from services in the Consul service mesh to
+services outside the mesh. These gateways effectively act as Connect proxies that can
+represent more than one service. They terminate Connect mTLS connections, enforce intentions,
+and forward requests to the appropriate destination.
+
+For data that must be encrypted in transit to the final destination, terminating gateways
+support TLS origination. By specifying a [CA file](/docs/agent/config-entries/terminating-gateway#cafile),
+[client certificate](/docs/agent/config-entries/terminating-gateway#certfile), and
+[private key](/docs/agent/config-entries/terminating-gateway#keyfile),
+connections from the gateway to a linked service can be encrypted using one-way or mutual TLS.
+If none of these are provided, Consul will only encrypt connections to the gateway and not
+from the gateway to the destination service. 
+
+-> **Note:** If certificates and keys are configured the gateway will upgrade HTTP connections to TLS. 
+Client applications can issue plain HTTP requests even when connecting to servers that require HTTPS.
+
+## Prerequisites
+
+Each terminating gateway needs two things:
+
+1. A local Consul agent to manage its configuration.
+2. General network connectivity to services within its local Consul datacenter.
+
+Terminating gateways also require that your Consul datacenters are configured correctly:
+
+- You'll need to use Consul version 1.8.0.
+- Consul [Connect](/docs/agent/options#connect) must be enabled on the datacenter's Consul servers.
+- [gRPC](/docs/agent/options#grpc_port) must be enabled on all client agents.
+- [Centralized configuration](/docs/agent/options#enable_central_service_config) must be enabled.
+
+Currently, [Envoy](https://www.envoyproxy.io/) is the only proxy with terminating gateway capabilities in Consul.
+
+- Terminating gateway proxies receive their configuration through Consul, which
+  automatically generates it based on the gateway's registration. Currently Consul
+  can only translate terminating gateway registration information into Envoy
+  configuration, therefore the proxies acting as terminating gateways must be Envoy.
+
+Sidecar proxies that send upstream traffic through a gateway aren't
```

Comment:
I prefer saying sidecar proxy when referring to the service proxy, however, we've been using the term Consul proxy or Connect proxy service. Maybe we can do a find/replace for both these docs so it's consistent?